components/tyk-dashboard/values.yaml

```diff
@@ -151,6 +151,8 @@ global:
   # It is used to set TYK_DB_HASHKEYS
   hashKeys: true
 
+  serviceAccountName: ""
```

Comment:
can you please elaborate why the field is located in `global` level? can't we move it to component level? maybe i am missing something